dev/tests/unit/Magento/FunctionalTestFramework/Console/BaseGenerateCommandTest.php

```diff
@@ -156,6 +156,24 @@ public function testThreeTestOneSuiteOneGroupMix()
         $this->assertEquals($expected, $actual);
     }
 
+    public function testSuiteToTestSyntax()
+    {
+        $testOne = new TestObject('Test1', [], [], []);
+        $suiteOne = new SuiteObject(
+            'Suite1',
+            ['Test1' => $testOne],
+            [],
+            []
+        );
+
+        $testArray = ['Test1' => $testOne];
+        $suiteArray = ['Suite1' => $suiteOne];
+        $this->mockHandlers($testArray, $suiteArray);
+        $actual = json_decode($this->callTestConfig(['Suite1:Test1']), true);
+        $expected = ['tests' => null, 'suites' => ['Suite1' => ['Test1']]];
+        $this->assertEquals($expected, $actual);
+    }
+
```

Comment:
Can we add negative tests for "Suite ${objectName} is not defined in xml" and "Cannot reference tests which are not declared as part of suite."?

Reply:
I'll give it a shot. I'm not 100% sure where that exception comes from (it may come from the manifest creation classes, which may mean a separate set of unit tests)

Reply:
Fixed